dependency/actions/compile/entrypoint/extensions/extensions2-8.1.json

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
Why is this named `extensions2`? Do we need this file? Also the extension file for 8.2 is missing